lib/ipmi_dcmi.c

```diff
@@ -3552,8 +1900,8 @@ ipmi_dcmi_parse_power(struct ipmi_intf * intf, int argc, char **argv)
 	case 0x02:
 		/* set limit */
 		if (argc < 4) {
-			print_strs(dcmi_pwrmgmt_set_usage_vals,
-			           "set_limit <parameter> <value>",
+			dcmi_print_strs(dcmi_pwrmgmt_set_usage_vals,
+			                "set_limit <parameter> <value>",
```

Comment:
you fixed one indentation change for the longer function name, but not the next one.  could be argued either way that it should also be changed.